web/frontend/src/components/camera/camera-manager.ts

```diff
@@ -31,7 +31,7 @@ export class CameraManager {
   open () {
     this.streamClient.add(this.cameraName);
     this.streamClient.on('track', (event) => {
-      const [eventStream] = event.streams;
+      const [eventStream] = (event as { streams: MediaStream[] }).streams;
```

Comment:
This is an issue max and I have noticed lately in TS SDK: callback args are being given `any` types. For now we can cast while investigating / fixing.